### PR TITLE
some small fixes that I found

### PR DESCRIPTION
**Pull request by dimaMachina**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,3 +36,5 @@ yarn-error.log*
 
 # typescript
 *.tsbuildinfo
+
+.idea/
```
Comment (B2o5T) [inline .gitignore:40]: to ignore my default webstorm settings folder

File: components/Paragraph.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ export const Paragraph = ({ children, ...props }: ParagraphProps) => {
       sx={{
         mt: Spacing['16px'],
         mb: Spacing['24px'],
-        '&:first-child': { mt: 0 },
+        '&:first-of-type': { mt: 0 },
```
Comment (B2o5T) [inline components/Paragraph.tsx:13]: image

File: pages/_app.tsx [hunk 3]
```diff
@@ -30,7 +30,7 @@ const DEFAULT_SEO_PROPS: DefaultSeoProps = {
   title: 'The Graph Docs',
   description: 'Browse the latest developer documentation including API reference, articles, and sample code',
   openGraph: {
-    site_name: 'The Graph Docs',
+    siteName: 'The Graph Docs',
```
Comment (B2o5T) [inline pages/_app.tsx:33]: image

**Review by saihaj — APPROVED** [review]

File: components/Paragraph.tsx (hunk 2 above)
Comment (benface) [reply components/Paragraph.tsx:13]: Hmm, these are not exactly equivalent, but I just tested and it doesn't break anything... nor does removing this entire rule... or the following one (the `&:last-child: { mb: 0 }` one). So, I propose doing that. 😄

File: components/Paragraph.tsx [hunk 4]
```diff
@@ -10,7 +10,7 @@ export const Paragraph = ({ children, ...props }: ParagraphProps) => {
...
         '&:last-child': { mb: 0 },
```
Comment (benface) [inline components/Paragraph.tsx:14]:
```suggestion
```

**Review by benface — APPROVED** [review]
Thank you!

**Pull request merged**